client/components/domains/domain-product-price/index.jsx

```diff
@@ -65,19 +65,24 @@ class DomainProductPrice extends React.Component {
 	renderIncludedInPremium() {
 		const { translate, showExpandedPrice } = this.props;
 
-		let textLabel = translate( 'Included in WordPress.com Premium' );
-
+		// TODO: When removing this flag, remember to remove PremiumPopover component
+		//       from the codebase.
 		const isKrackenUi = config.isEnabled( 'domains/kracken-ui' );
```

Comment:
For folks reading we can explicitly enable this feature flag with:
```
ENABLE_FEATURES=domains/kracken-ui npm start
```
And disable with:
```
DISABLE_FEATURES=domains/kracken-ui npm start
```

Reply:
TIL